### DISCUSSION TITLE
What's the difference between localhost:3000 and localhost:3000/alta?

### DISCUSSION THREAD

**Discussion by MarchandMD** [rubyforgood/pet-rescue#586] [Q&A] [answered]
As a newcomer to the project, it's very confusing to see one site displayed when I navigate to `localhost:3000` and a seemingly very different site at `localhost:3000/alta`. 

So very generally: what's the difference?

**Comment by jmilljr24** [comment, marked as answer]
This application supports multitenancy. Alta is just one of the organization names used for development. There is also seed data for an organization name of Baja. localhost:3000/alta and localhost:3000/baja will look the same minus their names and org specific data.

 The org specific home pages design was recently updated. That's the only reason it looks different from the root page at localhost:3000 (I'm not sure if/when that will be updated or what the planned use case for it is)

**Reply by MarchandMD** [reply]
Hmm, ok, this makes a little more sense. 

How would you feel about a PR with some additional language somewhere in the documentation to sort of clear this minor obfuscation? 

I appreciate your response and it helped organize things in my mind a little more; Maybe this additional clarity could help. 

Just a thought.

**Reply by kasugaijin** [reply]
Feel free to make a PR to add this tidbit to the ReadMe @MarchandMD. Maybe a couple lines with a link to the acts_as_tenant gem docs?
